src/utils.js

```diff
@@ -26,6 +26,21 @@ utils.remove0x = function(value) {
     }
 };
 
+/**
+ * Include 0x if prepended
+ * @param {String} value value to check and modify
+ * @return {String} string with 0x
+ */
+utils.include0x = function(value) {
```

Comment:
we were getting a double `0x` in certain circumstances

Reply:
oh i fixed this in another thing..but didn't fix it here haha. Did you update this or should we do this in a separate issue?

Reply:
do you mean update the issue / make a separate issue to address this? Or do you mean put this in a separate PR?

Reply:
Have you added this to this PR or are you going to make a separate issue?

Reply:
I added it to this PR. Just because it was a quick fix I made as I was working on the `to` value anyway. If you'd like me to separate it out and write an issue for it, lmk